db/document.go

```diff
@@ -1174,6 +1188,14 @@ func (doc *Document) UnmarshalWithXattr(ctx context.Context, data []byte, xdata
 			Cas: casOnlyMeta.Cas,
 		}
 		doc._rawBody = data
+	case DocUnmarshalVV:
+		tmpData := SyncData{}
```

Comment:
The motivation for these custom unmarshallers is to minimize the unmarshalling work.  In this case we're doing the same amount of work as DocUnmarshalNoHistory, so I think you could just use that.